elastic/integrations
# Create User risk score package

## What does this PR do?
Creates User risk score package by replicating the functionality of the previous [scripts](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/common/utils/risk_score_modules.ts).  


## Checklist


## How to test this PR locally

 
Followed similar steps used to test the Host risk score package:
- Use [elastic-package](https://github.com/elastic/elastic-package): elastic-package stack up -d -v --version 8.7.0-SNAPSHOT
- Go to Security -> Alerts -> Manage rules -> Create new rule: Create a rule that just generates a bunch of alerts (e.g. @timestamp: * and runs every 5 seconds). Make sure you have selected an index (e.g. `,etrics-*`) which has a value for `user.name` field. It is required to test this package.  
- Install the user risk score package with the UI or via elastic-package install
- Use dev tools to inspect whether the transforms are working as expected using following queries OR Go to  **Stack Management > Transforms** to check if they've started and producing results.
```
GET .alerts-security.user-risk-score-0.1.0
GET .alerts-security.user-risk-score-latest-0.1.0

GET .alerts-security.user-risk-score.all/_search
{
  "query": {
    "match_all": {}
  }
}

GET .alerts-security.user-risk-score-latest.all/_search
{
  "query": {
    "match_all": {}
  }
}
```

## Related issues
https://github.com/elastic/security-ml/issues/205


## Screenshots
Both pivot and latest transforms are installed and starting without any errors:
<img width="1468" alt="image" src="https://user-images.githubusercontent.com/109447885/215580654-c9c7acb0-a2d0-4a53-905b-9c4563d5232f.png">
